.dockerignore

```diff
@@ -0,0 +1 @@
+example
```

Comment:
Wat is dit?

Reply:
Dit zorgt ervoor dat de example folder niet in de docker buid context wordt opgenomen. Met name handig als je daar grote bestanden (bv gpkgs) hebt staan tijdens het testen.

Reply:
Nice, thanks voor de uitleg.